app/components/Navigation.tsx

```diff
@@ -27,42 +27,28 @@ import useDevice from "../hooks/useDevice";
 import useUser from "../hooks/useUser";
 import fetchAllLicenseTokens from "../utils/server/fetchAllLicenseTokens";
 import { useClient } from "urql";
-import colors from "../styles/colors";
+import useCurrentTheme from "../hooks/useCurrentTheme";
 
 const RootStack = createStackNavigator();
 const Stack = createStackNavigator();
 const Tab = createBottomTabNavigator();
 
-const theme = {
-  colors: {
-    primary: colors.text,
-  },
-};
-
-const nativePaperTheme = {
-  ...DefaultTheme,
-  colors: {
-    ...DefaultTheme.colors,
-    primary: colors.text,
-    accent: colors.textBrightest,
-  },
-};
-
-const headerOptions = {
```

Comment:
To avoid duplication we could create a function that generates the `headerOptions` based on the passed in theme or mode